### PR TITLE
feat: add nutmeg release support

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds nutmeg support:
- Update references from Maple to Nutmeg release
- Update AppArmor profile so it uses python > 3 (see [ref](https://github.com/eduNEXT/tutor-contrib-codejail/issues/19))

### How to test
1. Install tutor, tutor-contrib-codejail, [codejailservice](https://github.com/eduNEXT/codejailservice) from source
2. Move to nutmeg branches: nutmeg -for tutor-, 14.0.0 -for codejailservice-, MJG/nutmeg -for tutor-contrib-codejail-
3. Install plugin
4. Build images: tutor config save && tutor images build all
5. Init services: tutor dev init
6. Follow test instructions [here](https://github.com/eduNEXT/tutor-contrib-codejail#how-to-know-if-codejail-is-working). The error will appear in the course component.

### CODE REVIEW

File: tutorcodejail/templates/codejail/build/codejail/Dockerfile
Comment (felipemontoya) [inline tutorcodejail/templates/codejail/build/codejail/Dockerfile:54]: Do we not need symmath anymore? or is it included in py38?

Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile:54]: That path was removed in nutmeg: https://github.com/openedx/edx-platform/commit/8e8d8404bed130c071a331da8f3a1247d8cebf00
https://github.com/openedx/edx-platform/tree/open-release/nutmeg.master/common/lib

File: tutorcodejail/plugin.py
Comment (mariajgrimaldi) [inline tutorcodejail/plugin.py:20]: We changed it from 3.5 to 3.8 since the sandbox requirements didn't include py3.5 anymore. 
Now, which patch version should we use here? 
We first tried to use 3.8.6 but then changed it to 3.8.5 so it doesn't crash with the codejailservice pyenv (which also uses 3.8.6).
@felipemontoya @Alec4r @MoisesGSalas @jfavellar90 @johanv26

Comment (MoisesGSalas) [reply tutorcodejail/plugin.py:20]: maybe you can use [pyenv-alias](https://github.com/s1341/pyenv-alias) and install the sandbox version as `x.y.z_sandbox` ?

File: tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox
Comment (MoisesGSalas) [inline tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox]: Wasn't this intended for the sandbox version only? you can use `{{ SANDBOX_PYTHON_VERSION }}` right?

File: tutorcodejail/templates/codejail/build/codejail/Dockerfile
Comment (felipemontoya) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile:54]: I see.

Now that it is located at openedx-calc
> Symmath has been added to openedx-calc alongside calc in https://github.com/openedx/openedx-calc/pull/38.

Do we need to allow it or install it or something else?

File: tutorcodejail/plugin.py
Comment (mariajgrimaldi) [reply tutorcodejail/plugin.py:20]: That's a great suggestion! Thanks, I'll try that

File: tutorcodejail/templates/codejail/build/codejail/Dockerfile
Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile:54]: As you said, it's now in py38.txt

File: tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox
Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox]: That's right! Thanks

**Comment by mariajgrimaldi** [comment]
Can you review again please? @MoisesGSalas @felipemontoya

File: tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox
Comment (MoisesGSalas) [inline tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox]: This should be `{{ CODEJAIL_SANDBOX_PYTHON_VERSION }}_sandbox`

File: tutorcodejail/templates/codejail/build/codejail/Dockerfile
Comment (MoisesGSalas) [inline tutorcodejail/templates/codejail/build/codejail/Dockerfile]:
```suggestion
RUN $PYENV_ROOT/versions/"$SANDBOX_PYTHON_VERSION"_sandbox/bin/python -m venv --copies /sandbox/venv
```

File: tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox
Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/apps/profiles/docker-edx-sandbox]: I totally missed that. Thanks

File: tutorcodejail/templates/codejail/build/codejail/Dockerfile
Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile]: Good, thanks. I'll build the image and let you know how it goes

**Comment by DeanJayMathew** [comment]
@mariajgrimaldi  Has this fix been successful?

The reason I ask is because it is a blocker to resolving this Nutmeg testing issue: https://github.com/openedx/build-test-release-wg/issues/159

**Comment by mariajgrimaldi** [comment]
Yes @DeanJayMathew. I'm just waiting for an approval to merge and release.

**Comment by mariajgrimaldi** [comment]
I've tested building a new image with the latest changes in the PR and it's working!
image
image

Using the course attached [here](https://github.com/eduNEXT/tutor-contrib-codejail/blob/main/docs/resources/course_codejail_example.tar.gz)

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**
